packages/oui-modal/src/modal.controller.js

```diff
@@ -0,0 +1,2 @@
+export default class ModalController {
+}
```

Comment:
Empty controller, is it necessary in a separate file ?

Reply:
I created it without thinking about it until I realized that it was useless :D ... but I forgot to remove it :/